core/src/main/java/org/mule/runtime/core/internal/message/ErrorBuilder.java

```diff
@@ -212,17 +262,19 @@ public Error build() {
     private final ErrorType errorType;
     private final Message muleMessage;
     private final List<Error> errors;
+    private final List<Error> suppressedErrors;
```

Comment:
this will have to be considered in DataSense tests

Reply:
I'm currently working in the datasense api modifications